index.html

```diff
@@ -279,9 +279,15 @@ <h2>Contact me</h2>
               love to help with it.
             </p>
             <div class="tonic-courses">
```

Comment:
- [ ] OPTIONAL: I have noticed that you are using "grid" to layout the elements, but based on the requirements you should be using `flexbox`, what about giving try to  `flexbox` ? 😉 (This requirement is optional, because grid works just fine!)